.github/workflows/unit-tests.yml

```diff
@@ -44,13 +44,12 @@ jobs:
     - name: Select Xcode
       run: sudo xcode-select -s /Applications/Xcode_14.1.app/Contents/Developer
 
+    - name: Install xcbeautify
+      continue-on-error: true
+      run: brew install xcbeautify
+
     - name: Run tests
-      # BSK does not support running tests in parallel, but the flag is required
-      # to make XUnit output work. Making it parallel with 1 worker then.
-      # https://stackoverflow.com/a/70040836
-      run: |
-        swift test --parallel --num-workers=1 \
-          --xunit-output tests.xml
+      run: swift test | xcbeautify --report junit --report-path . --junit-report-filename tests.xml
```

Comment:
I’ve pushed a commit that breaks tests and it still managed to report success.
This may be because xcbeautify overrifing the non-zero exit status code of swift test.
To propagate status code down the Unix pipe, use pipefail modifier such as:

```bash
set -o pipefail && swift test | xcbeautify ...
```

Reply:
Thank you for spotting that 👏🏼 ! I only tested failed tests and success, but I didn't test if tests didn't compile.

I will add the command to propagate the status code, and I will test the different scenarios and let you know